package.json

```diff
@@ -28,9 +31,15 @@
     "@nrwl/linter": "13.4.5",
     "@nrwl/node": "13.4.5",
     "@nrwl/react": "13.4.5",
+    "@nrwl/storybook": "^13.7.2",
```

Comment:
```suggestion
    "@nrwl/storybook": "13.4.5",
```

Can you please downgrade `@nrwl/storybook` to the same version? As the package uses strict dependencies this adds a lot of duplicates:

![image](https://user-images.githubusercontent.com/14183168/152786887-acf3c775-88f5-4141-a77c-fd0d963eb4da.png)

Reply:
Downgraded, but even with that, there are +6000 lines in yarn.lock :/

Reply:
Can you please also run `yarn dedupe`? (it's built into Yarn Berry)